src/VisionModel/resnet.jl

```diff
@@ -0,0 +1,82 @@
+using Flux
+
+basicblock(inplanes::Int, outchannels, downsample::Bool = false) = downsample ? 
+  Chain(Conv((3, 3), inplanes => outchannels[1], stride = 2),
+        BatchNorm(inplanes, λ = relu),
+        Conv((3, 3), inplanes => outchannels[2], stride = 1, pad = 1),
+        BatchNorm(outchannels, λ=relu)) : 
+  Chain(Conv((3, 3), inplanes => outchannels[1], stride = 1),
+        BatchNorm(inplanes, λ = relu),
+        Conv((3, 3), inplanes => outchannels[2], stride = 1, pad = 1),
+        BatchNorm(outchannels, λ=relu))
+
+bottleneck(inplanes::Int, outchannels, downsample::Bool = false) = downsample ?
+  Chain(Conv((1, 1), inplanes => outchannels[1], stride = 2),
+        BatchNorm(inplanes, λ = relu),
+        Conv((3, 3), inplanes => outchannels[2], stride = 1, pad = 1),
+        BatchNorm(inplanes, λ = relu),
+        Conv((1, 1), inplanes => outchannels[3], stride = 1),
+        BatchNorm(outchannels, λ = relu)) :
+  Chain(Conv((1, 1), inplanes => outchannels[1], stride = 1),
+        BatchNorm(inplanes, λ = relu),
+        Conv((3, 3), inplanes => outchannels[2], stride = 1, pad = 1),
+        BatchNorm(inplanes, λ = relu),
+        Conv((1, 1), inplanes => outchannels[3], stride = 1),
+        BatchNorm(outchannels, λ = relu))
```

Comment:
I made a slight error in my original code. Can you take a look at the edited comment and apply those changes? Mainly, it should not be `inplanes => outchannels[2]`. It should be `outchannels[1] => outchannels[2]`. Same for the # of channels in batch norm, etc.